### ISSUE TITLE
NVDA fails to read some text in Word since 2016/05/25's Next

### ISSUE THREAD

**Issue by sukiletxe** [nvaccess/nvda#5999]
STR:
1. Open Word (tested with Word 2007)
2. Type some text.
3. Read it.

An error is triggered and no text is read.
[Log.txt](https://github.com/nvaccess/nvda/files/283924/Log.txt)

**Comment by sukiletxe** [comment]
[Log.txt](https://github.com/nvaccess/nvda/files/283926/Log.txt)

**Comment by michaelDCurran** [comment]
Broken and then fixed by #2961
